.gitignore

```diff
@@ -15,3 +15,4 @@ debug.out
 rusty-tags.vi
 *.sw*
 tags
+.direnv
```

Comment:
Is this a directory that gets added to the working copy due to this PR?

Reply:
yes.

```
vi@philomena ~/o/omicron (eliza/nixos-build)> ls .direnv/
flake-profile@  flake-profile-1-link@
```

Reply:
Yes, if using the flake via direnv, a bunch of stuff gets cached in `.direnv`.